cmd/run_test.go

```diff
@@ -149,6 +151,46 @@ func TestInitRuntimeVerifyNonBundle(t *testing.T) {
 	}
 }
 
+func TestInitRuntimeCipherSuites(t *testing.T) {
```

Comment:
Only a small nit: I often find tests easier to read, maintain, and parse failure output from if they're parameterized. This test could for instance be broken up into a couple of test cases that we iterate over.

Reply:
Good point. Converted this to a parameterized test.